### PR TITLE
Fix sbt publish

### PR DESCRIPTION
**Pull request by jxnu-liguobin**
Automatically publish sbt plugin and remove some kotlin in sbt.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -203,9 +203,15 @@ jobs:
...
+        uses: olafurpg/setup-scala@v10
         if: github.event.inputs.publish_sbt_plugin == 'Y'
         working-directory: plugins/sbt/graphql-java-codegen-sbt-plugin
-        run: sbt release with-defaults default-tag-exists-answer k
+        run: sbt ci-release
+        env:
+          PGP_PASSPHRASE: ${{ secrets.PGP_PASSPHRASE }}
+          PGP_SECRET: ${{ secrets.PGP_SECRET }}
+          SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+          SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
```
Comment (jxnu-liguobin) [inline .github/workflows/release.yml:214]: add to 
image

**Review by kobylynskyi — APPROVED** [review]

**Comment by kobylynskyi** [comment]
@jxnu-liguobin approved. Can I merge this?

**Comment by jxnu-liguobin** [comment]
> @jxnu-liguobin approved. Can I merge this?

yes , I sent an email a long time ago, and OSS account is in the email.

**Comment by kobylynskyi** [comment]
> > @jxnu-liguobin approved. Can I merge this?
> 
> yes , I sent an email a long time ago, and OSS account is in the email.

which email are you referring to?

**Comment by kobylynskyi** [comment]
This was released in 5.10.0. Thanks!

**Pull request merged**